sources/android-gamesdk/GameActivity/game-activity/include/game-activity/native_app_glue/android_native_app_glue.c

```diff
@@ -102,7 +105,6 @@ void android_app_pre_exec_cmd(struct android_app* android_app, int8_t cmd) {
 
         case APP_CMD_TERM_WINDOW:
             LOGV("APP_CMD_TERM_WINDOW");
-            pthread_cond_broadcast(&android_app->cond);
```

Comment:
Why remove this line?

Reply:
The calling of pthread_cond_broadcast will increase the occurrence probability of anr

Reply:
pthread_cond_broadcast called here maybe increase the occurrence probability of anr.
It may cause the APP_CMD_TERM_WINDOW processing in android_app_post_exec_cmd to complete before __futex_wait_ex(called from pthread_cond_wait)